content/docs/components/switch/theming.mdx

```diff
@@ -2,3 +2,157 @@
 id: switch
 scope: theming
 ---
+
+The `Switch` component is a multipart component. The styling needs to be applied
+to each part specifically.
+
+> To learn more about styling multipart components, visit the
+> [Component Style](/docs/styled-system/component-style#styling-multipart-components)
+> page.
+
+## Anatomy
+
+- A: `container`
+- B: `thumb`
+- C: `track`
```

Comment:
I've prepared a custom anatomy illustration for you. Place [this code](https://gist.github.com/nikolovlazar/28bda596bbb1fa8a2ec42f097c4ff3e0) below the anatomy list.

Reply:
Added it on https://github.com/chakra-ui/chakra-ui-docs/pull/1117/commits/9acf3dc59cea08fe18bd838d3c84b2dd0bf32bea